plugins/dev-tools/src/index.js

```diff
@@ -6,14 +6,23 @@
 
 import toolboxCategories from './toolboxCategories';
 import toolboxSimple from './toolboxSimple';
-import addGUIControls from './addGUIControls';
+let addGUIControls;
+if (typeof window !== 'undefined') {
+  addGUIControls = require('./addGUIControls').default;
+}
+import {assertFieldValue, runConstructorSuiteTests, runFromJsonSuiteTests,
```

Comment:
Can we just export all of these under a single namespace? eg: 
```js
import * as testHelpers from './field_test_helpers.mocha';

export {
 ....
 testHelpers,
 ....
}
```
and then in the test code: 
```js
import {testHelpers} from '@blockly/dev-tools';
const {assertFieldValue, runConstructorSuiteTests, runFromJsonSuiteTests, runSetValueTests} = testHelpers;

```

Just in an effort of more grouping, and reduction of clutter.

Reply:
Updated in dev-tools. I've updated the requires in the test code as well to use the exported testHelpers (since we don't currently support es6 modules for the test code).